lib/CodeTransform/Adapter/WorseReflection/Refactor/WorseExpandClass.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Phpactor\CodeTransform\Adapter\WorseReflection\Refactor;
+
+use Microsoft\PhpParser\Node\QualifiedName;
+use Microsoft\PhpParser\Parser;
+use Phpactor\CodeTransform\Domain\Refactor\ExpandClass;
+use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\Extension\SourceCodeFilesystemExtra\SourceCodeFilestem\Application\ClassSearch;
+use Phpactor\TextDocument\TextDocumentEdits;
+use Phpactor\TextDocument\TextEdit;
+use Phpactor\TextDocument\TextEdits;
+use Phpactor\WorseReflection\Reflector;
+
+class WorseExpandClass implements ExpandClass
+{
+    private Reflector $reflector;
+
+    private ClassSearch $classSearch;
+
+    private Parser $parser;
+
+    public function __construct(
+        Reflector $reflector,
+        ClassSearch $classSearch,
+        Parser $parser = null
+    ) {
+        $this->reflector = $reflector;
+        $this->classSearch = $classSearch;
+        $this->parser = $parser ?: new Parser();
+    }
+
+    public function getTextEdits(SourceCode $sourceCode, int $offset): TextDocumentEdits
+    {
+        $symbolInformation = $this->reflector
+            ->reflectOffset($sourceCode->__toString(), $offset)
+            ->symbolContext()
+            ->symbol();
+
+        $result = $this->classSearch->classSearch('filesystem', $symbolInformation->name());
```

Comment:
This is probably not the best idea. I think we should be able to look through the use statements and see if any class name there matches. What do you think?